src/main/java/org/simulator/examples/SimulatorExample.java

```diff
@@ -31,78 +31,96 @@
 import javax.xml.stream.XMLStreamException;
 
 import org.apache.commons.math.ode.DerivativeException;
+import org.apache.log4j.Logger;
 import org.jfree.ui.RefineryUtilities;
 import org.sbml.jsbml.Model;
 import org.sbml.jsbml.SBMLDocument;
 import org.sbml.jsbml.SBMLException;
 import org.sbml.jsbml.validator.ModelOverdeterminedException;
 import org.sbml.jsbml.SBMLReader;
+import org.simulator.comp.CompSimulator;
 import org.simulator.math.odes.*;
 import org.simulator.plot.PlotMultiTable;
 import org.simulator.sbml.SBMLinterpreter;
 import org.simulator.plot.PlotMultiTable;
 
 /**
  * A simple program that performs a simulation of a model.
- * 
+ *
  * @author Andreas Dr&auml;ger
  * @version $Rev$
  * @since 0.9
  */
 public class SimulatorExample {
 
-  /**
-   * Starts a simulation at the command line.
-   * 
-   * @param args
-   *        file name, step size, and end time.
-   * @throws IOException
-   * @throws XMLStreamException
-   * @throws SBMLException
-   * @throws ModelOverdeterminedException
-   * @throws DerivativeException
-   */
-  public static void main(String[] args) throws XMLStreamException,
-  IOException, ModelOverdeterminedException, SBMLException,
-  DerivativeException {
+    private static Logger logger = Logger.getLogger(SimulatorExample.class.getName());
+    private static final double TOLERANCE_FACTOR = 1E-3;
+    private static final int WIDTH = 400;
+    private static final int HEIGHT = 400;
 
-    // Configuration
-    String fileName = args[0];
-    double stepSize = Double.parseDouble(args[1]);
-    double timeEnd = Double.parseDouble(args[2]);
-    double absTol = Double.parseDouble(args[3]);
-    double relTol = Double.parseDouble(args[4]);
+    /**
+     * Starts a simulation at the command line.
+     *
+     * @param args file name, step size, and end time.
+     * @throws IOException
+     * @throws XMLStreamException
+     * @throws SBMLException
+     * @throws ModelOverdeterminedException
+     * @throws DerivativeException
+     */
+    public static void main(String[] args) throws XMLStreamException,
+            IOException, ModelOverdeterminedException, SBMLException,
+            DerivativeException {
 
-    // Read the model and initialize solver
-    SBMLDocument document = (new SBMLReader()).readSBML(fileName);
-    Model model = document.getModel();
-    
-    DESSolver solver = new RosenbrockSolver();
-    solver.setStepSize(stepSize);
-    SBMLinterpreter interpreter = new SBMLinterpreter(model);
-    if (solver instanceof AbstractDESSolver) {
-      ((AbstractDESSolver) solver).setIncludeIntermediates(false);
-    }
+        String fileName = "";
```

Comment:
Normally one initializes with `null`. Empty string is always a very risky choice. Change to `null`.

Reply:
Updated the code!